tools/lxdclient/client_network.go

```diff
@@ -49,13 +49,33 @@ type creator interface {
 	ProfileConfig(profile string) (*shared.ProfileConfig, error)
 }
 
+func checkBridgeConfig(client rawNetworkClient, bridge string) error {
+	n, err := client.NetworkGet(bridge)
+	if err != nil {
+		return err
+	}
+
+	if n.Config["ipv6.address"] != "none" {
+		return errors.Errorf(`juju doesn't support ipv6. Please disable LXD's IPV6:
```

Comment:
I guess I'm slightly confused. This isn't about ipv6 only: the default LXD network has ipv4 connectivity. I thought the whole point was to disable ipv6 entirely in all cases, because it wasn't supported. The error message you've suggested makes me think that it should work in this case, since the LXD bridge has ipv4 connectivity too.